cli.js

```diff
@@ -125,6 +125,14 @@ const handleExportCommand = async args => {
   await ach.export(doctypes, filename, last)
 }
 
+const handleExportSingleCommand = async args => {
+  let { doctype, id, filename, url, token, last } = args
+  token = token || autotoken(url, [doctype])
+  const ach = new ACH(token, url, [doctype])
+  await ach.connect()
+  await ach.exportSingle(doctype, id, filename, last)
```

Comment:
`last` arg seems not to be used by `exportDocs()`

Reply:
Good catch!